lib/read.mll

```diff
@@ -1143,30 +1143,30 @@ and junk = parse
 
   exception Finally of exn * exn
 
-  let stream_from_lexbuf v ?(fin = fun () -> ()) lexbuf =
+  let seq_from_lexbuf v ?(fin = fun () -> ()) lexbuf =
     let stream = Some true in
-    let f i =
-      try Some (from_lexbuf v ?stream lexbuf)
+    let rec f () =
+      try Seq.Cons(from_lexbuf v ?stream lexbuf, f)
```

Comment:
```suggestion
      try Seq.Cons (from_lexbuf v ?stream lexbuf, f)
```

(low-key wishing we could use automatic code formatting so I wouldn't complain about such petty things).

Reply:
Done!